src/Bicep.LangServer/Completions/BicepCompletionContext.cs

```diff
@@ -373,6 +373,18 @@ private static BicepCompletionContextKind GetDeclarationTypeFlags(IList<SyntaxBa
                 SyntaxMatcher.IsTailMatch<ModuleDeclarationSyntax, StringSyntax, Token>(matchingNodes, (_, _, token) => token.Type == TokenType.StringComplete) ||
                 SyntaxMatcher.IsTailMatch<ModuleDeclarationSyntax, SkippedTriviaSyntax, Token>(matchingNodes, (module, skipped, _) => module.Path == skipped))
             {
+                if (matchingNodes.Count == 4 &&
+                    matchingNodes[^1] is Token token &&
+                    token.Type == TokenType.StringComplete)
+                {
+                    string text = token.Text;
+
+                    if (text.StartsWith("'br/") || text.StartsWith("'br:"))
```

Comment:
>br/ [](http://example.com/codeflow?start=42&length=3)

A relative path like `br/main.bicep` would also match the condition but would not be an OCI reference.

Reply:
Removed this as part moving logic to CompletionProvider:
https://github.com/Azure/bicep/commit/a699dbfc721de8ace5d95d935c541cfa51959a35